utils/src/main/scala/cromwell/pipeline/utils/configs/LogInfoUtils.scala

```diff
@@ -0,0 +1,84 @@
+package cromwell.pipeline.utils.configs
+
+import cromwell.pipeline.utils._
+import cromwell.pipeline.utils.configs.SecretData._
+import play.api.libs.functional.syntax.{ toFunctionalBuilderOps, unlift }
+import play.api.libs.json.Format.GenericFormat
+import play.api.libs.json.{ Json, Writes, __ }
+
+trait LogInfoUtils {
+  def getConfigMessage: String
+}
```

Comment:
Why do you need this trait?

Reply:
As far as I understand from our mentors explanation this may be useful for future functionality expansion and may be a good practice in general. Quite possible I got it wrong though.

Reply:
Well, it might be, but you need to use it. 
Right now if you remove this trait it will change nothing. It won't even request any code changes.
We should not have anything we're not using in our codebase.

Reply:
@ilyaonishenko @Nikolai-Isaev 
Let me explain why asked to add this trait.
It's quite hard to test classes that use classic `Utils` objects. I.e.
```
object Utils {
  def foo(n: Int): Unit = println(n)
}
class A {
  def hardToTest(str: String): Int = {
    val i = str.toInt
    Utils.foo(i)
    i
  }
}
```
It's tricky to verify that `hardToTest` called `foo` method if we want to write unit test.
When we use utils objetcs, we bypass DI and rely on some global state. It is no better than
```
class Utils {
  def foo(n: Int): Unit = println(n)
}
class A {
  private val utils = new Utils
  def hardToTest(str: String): Int = {
    val i = str.toInt
    utils.foo(i)
    i
  }
}
```
That's why I asked to add a separate trait for this `utils` class. If we use trait, then it will be easily testable and reusable
```
trait Utils {
  def foo(n: Int): Unit
}
object Utils extends Utils {
  def foo(n: Int): Unit = println(n)
}
class A(utils: Utils) {
  def hardToTest(str: String): Int = {
    val i = str.toInt
    utils.foo(i)
    i
  }
}
```
And we still can use it as classic `Utils` class if required.
In this particular case it won't make much of a difference, but it might be useful in the future and a good starting point for getting rid of classic `utils`. You are free to drop it if it casuses too many problems.

Reply:
@myazinn, that's could be a case, if `LogInfoUtils` extends this trait. :)
But right now this trait just exists nearby. 
@Nikolai-Isaev, you need to make LogInfoUtils to use this trait.

Reply:
Done